absl/crc/internal/cpu_detect.cc

```diff
@@ -269,8 +269,12 @@ CpuType GetCpuType() {
 }
 
 bool SupportsArmCRC32PMULL() {
+#if defined(HWCAP_CRC32) && defined(HWCAP_CRC32PMULL)
```

Comment:
Are you sure `HWCAP_CRC32PMULL` isn't a typo for `HWCAP_PMULL`?

Reply:
Sorry, I made mistake it must be `HWCAP_PMULL`